Samples/Islands/cs-winui-packaged/WinUIDesktopAppWithIsland/README.md

```diff
@@ -0,0 +1,13 @@
+
+> NOTE: This sample uses Experimental APIs
+
+This sample shows how you can host a ContentIsland in a WinUI Desktop app.
+
+When you click the "Add Helmet" button, it creates a Scene Node island that loads the 3d helmet model and sets up the 3d scene.
```

Comment:
Just like at Build 2024, can you also add Lottie? Since this is in experimental APIs, it's okay if we need a little time to sort through the Lottie-side PR as well.

Reply:
Done!